### ISSUE TITLE
Do Not Scan Hidden Folders

### ISSUE THREAD

**Issue by mark-monteiro** [grimsi/gameyfin#43]
Hidden folders (i.e. folders with a filename that starts with a dot) should be ignored from the scanning process. This would also fix the issue with the `.gameyfin` directory showing up as an un-mapped file.

**Comment by grimsi** [comment]
Yup, I actually wanted to do this and just forgot about it. Thanks for reminding me!

**Comment by mark-monteiro** [comment]
No problem, I figured something like this might get lost under all the new feature requests 😅 

It might also be worth looking into [point number 2 here](https://github.com/grimsi/gameyfin/issues/37#issuecomment-1272847674) (ignore empty files/folders during scan) while working on this, since they're pretty closely related

**Comment by grimsi** [comment]
Fixed in #44
